build/azure-pipelines/build-common.yml

```diff
@@ -12,6 +12,23 @@ steps:
 - task: GoTool@0
   inputs:
     version: '1.16.5'
+    goBin: $(Build.SourcesDirectory)
+
+- task: Go@0
+  displayName: 'Go install go-winres'
+  inputs:
+    command: 'custom'
+    customCommand: 'install'
+    arguments: 'github.com/tc-hib/go-winres@latest'
+    workingDirectory: '$(Build.SourcesDirectory)/cmd/sqlcmd'
+  env:
+      GOBIN: $(Build.SourcesDirectory)
+
+- task: CmdLine@2

```

Comment:
I tried to use `go generate` instead but couldn't get ADO `task.prependpath` to add GOBIN to the path successfully on Windows.